terraform/live/rabbitmq/aws/terragrunt.hcl

```diff
@@ -0,0 +1,61 @@
+locals {
+  cluster_name = "mqperf-cluster"
+  region       = "eu-central-1"
+}
+
+remote_state {
+  backend = "s3"
+  config = {
+    encrypt        = true
+    bucket         = get_env("TF_VAR_BUCKET_NAME")
+    key            = "${path_relative_to_include()}/terraform.tfstate"
+    region         = get_env("TF_VAR_AWS_REGION", "eu-central-1")
+    dynamodb_table = get_env("TF_VAR_DYNAMODB_TABLE")
+  }
+  generate = {
+    path      = "backend.tf"
+    if_exists = "overwrite_terragrunt"
+  }
+}
+
+generate "provider" {
+  path      = "provider.tf"
+  if_exists = "overwrite_terragrunt"
+  contents  = <<EOF
+terraform {
+  required_providers {
+    aws = {
+      version = ">= 4.15.1"
+      source = "hashicorp/aws"
+    } 
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.12.1"
+    }
+
+
+  }
+}
+
+data "aws_eks_cluster" "eks" {
+        name = "mqperf-cluster"
```

Comment:
I think we should do this like this for now `name = ${get_env("CLUSTER_NAME", locals.cluster_name)}` so it will work from setup script and if run with terragrunt by hand it will load cluster name from locals